app/views/appearances/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :page_header do %>
+  <h1><span class="fa fa-paint-brush"></span> <%= t(:'sufia.admin.sidebar.appearance') %></h1>
+<% end %>
+
+<div class="row">
+  <div class="col-md-12">
+    <div class="panel panel-default">
+      <%= form_for(@site, url: site_appearances_path) do |f| %>
+        <div class="panel-heading">
+          <h3 class="panel-title">Site Appearance</h3>
```

Comment:
The `<h1>` above it is already using the i18n string `sufia.admin.sidebar.appearance`, which is why I didn't reuse it for this `<h3 class="panel-title">`.  I agree though that more of Lerna should use i18n strings. But, in this situation I was following the same structure as other "edit" pages (e.g. [`content_blocks/edit.html.erb`](https://github.com/projecthydra-labs/lerna/blob/master/app/views/content_blocks/edit.html.erb) and [`labels/edit.html.erb`](https://github.com/projecthydra-labs/lerna/blob/master/app/views/labels/edit.html.erb))